### ISSUE TITLE
Add Coinkite

### ISSUE THREAD

**Issue by nvk** [litecoin-project/litecoin.org-ancient#7]
Hello,

We would love to be listed on Litecoin.org.

AFIK we are the only large Litecoin web wallet for a while now, I thought i'd ask to be included on the website. 

We are also on bitcoin.org already.

PS: Litecoin fast blocks are awesome for retail payments!

**Comment by doc-hex** [comment]
:+1: biased

**Comment by wtogami** [comment]
litecoin.org will be redone from scratch soon.  It would be best to include such links with appropriate warning like bitcoin.org when the site is redone.

**Comment by nvk** [comment]
@wtogami I agree, would be good to add something explaining what it is to not cause confusion.
